test/onnx/test_pytorch_onnx_shape_inference.py

```diff
@@ -268,6 +271,60 @@ def test_resize_after_concat(self):
         )
         self.run_test(g, resize.node(), expect_tensor("Float", shape=(4, 32, 128, 128)))
 
+    def test_autograd_layernorm_shape(self):
+        def symbolic_pythonop(ctx: torch.onnx.SymbolicContext, g, *args, **kwargs):
+            return g.op("com.microsoft::PythonOp")
+
+        torch.onnx.register_custom_op_symbolic("prim::PythonOp", symbolic_pythonop, 1)
+        self.addCleanup(torch.onnx.unregister_custom_op_symbolic, "prim::PythonOp", 1)
+
+        # necessay parameters for transformer embeddings
+        hidden_size, max_position_embeddings, batch_size = (
```

Comment:
nit: readability: Assign one variable on each line

Reply:
Also move this down to just before when they are used?

Reply:
hidden_size is consumed in autograd.func right after this.